src/components/header/header.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, HStack, VStack } from "@nypl/design-system-react-components";
+import React, { useEffect, useState } from "react";
+import DCLogo from "../logo/logo";
+
+const Header = () => {
+  const [isScrolled, setIsScrolled] = useState(true);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      if (window.scrollY > 100) {
```

Comment:
How did you calculate the number 100?

Reply:
Vibes

Reply:
Custom hook uses `offsetHeight` which seems more legitimate but open to discuss